backend/django/core/views/api_annotate.py

```diff
@@ -926,3 +942,23 @@ def get_label_history(request, project_pk):
         results.append(temp_dict)
 
     return Response({"data": results})
+
+
+@api_view(["POST"])
+# @permission_classes((IsCoder,))
+def modify_metadata_value(request, metadata_pk):
```

Comment:
So the only issue I'm seeing with this is that you have to click away from the history tab and back for the change to show up when you maximize the item. 

Not really a problem though I think? Just something the users have to be aware of. In general I'd add a line to the instructions on the top of the page. Something like "In this table you may edit metadata fields. Click on the value in the column and row where you want to change the data and it will open as a text box."